README.md

```diff
@@ -153,173 +167,115 @@ Wiphy phy13
 	max scan plan interval: -1
 	max scan plan iterations: 0
 	Supported extended features:
-Wiphy phy14
-	... (omit)
-Wiphy phy15
-	... (omit)
 ```
 
-The "managed mode" in the Supported interface modes is identical to station mode.
-
-### Getting Station Information
+You can see the supported operating modes, supported ciphers, channels, bitrates, and supported commands in the output.
 
-To retrieve station information for `owl0`, execute the following command:
-```shell
-$ sudo iw dev owl0 station get 00:6f:77:6c:30:00
-```
-
-You should see output similar to the following:
-```
-Station 00:6f:77:6c:30:00 (on owl0)
-	inactive time:	600260 ms
-	rx bytes:	0
-	rx packets:	0
-	tx bytes:	0
-	tx packets:	0
-	tx failed:	0
-	signal:  	-33 dBm
-	current time:	1655310275763 ms
-```
+The "managed mode" in the Supported interface modes is identical to station mode.
 
 ### Creating Network Namespaces
+In our testing environment, we have two network interfaces: `owl0` in HostAP mode and `owl1` in station mode. Thus, we need only one network namespace in order to isolate them.
 
-Next, create three network namespaces using the following commands:
+Create a network namespace using the following commands:
```

Comment:
a network namespace?

Reply:
Since I have simplify the testing environment to be only one STA and one HostAP, so we just need one network namespace (one remains in `init_net` is fine). Do you think we should remain the two STA and one HostAP testing environment?

Reply:
I expect the minimal changes against the existing code when a new feature comes in.